tools/c7n_azure/requirements.txt

```diff
@@ -1,4 +1,4 @@
-vcrpy==2.0.1
+vcrpy==2.1.0
```

Comment:
I had thought 2.1 wouldn't work but the build failures I'm seeing look unrelated (just a regex issue) - so hoping the issue was resolved in newer Azure SDK's.